src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You're adding a BOM here. Did you use VS to edit this file? Can you set the encoding to be without BOM?

Reply:
our editorconfig does set utf-8 (which means without BOM), but VS just doesn't respect that... 😠

Reply:
the linter we use is unfortunately not able to catch charset violations
https://github.com/editorconfig-checker/editorconfig-checker/issues/85